modules/auth/cms_authentication_module.e

```diff
@@ -125,6 +126,29 @@ feature -- Hooks configuration
 
 feature -- Handler
 
+	handle_account (api: CMS_API; req: WSF_REQUEST; res: WSF_RESPONSE)
+		local
+			r: CMS_RESPONSE
+		do
+			create {GENERIC_VIEW_CMS_RESPONSE} r.make (req, res, api)
+			r.set_value ("Account Info", "optional_content_type")
```

Comment:
better not setting an optional_content_type here .. since there is no content_type  "Account info"